tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore

```diff
@@ -1,7 +1,7 @@
 <article class="course" role="region" aria-label="<%- content.display_name %>">
         <header class="course-image">
             <div class="cover-image">
-                <img src="<%- image_url %>" onerror="this.src='/static/indigo/images/no_course_image.png';" alt="<%- content.display_name %> <%- content.number %>" />
+                <img src="<%- image_url %>" onerror="this.src='${static.url('images/no_course_image.png')};" alt="<%- content.display_name %> <%- content.number %>" />
```

Comment:
Hi, @misilot. This is not working fine as the file's extension is `underscore` and its a `mako` template. It doesn't support `static.url` that's why I have to hardcode the path. The above change in `course.html` is fine as its a `html` template. Can you please test it on your system?

Reply:
Sure, can you give me an example of where I can find this in the LMS? As the two places I thought they might be, it _was_ working.

I can update it to `/theming/asset/images/no_course_image.png` if that would make more sense, so it isn't hardcoded to the specific theme?

Reply:
I also haven't an example of this in LMS. In the first place, it's perfectly working. In second case `course_card.underscore`, it's redering static.url as string.

We can update to `/theming/asset/images/no_course_image.png` (as it's working fine), but the problem is `no_course_image.png` currently exists in only indigo theme. Lets update to `/theming/asset/images/no_course_image.png`.

Reply:
I've gone and update it to be `/theming/asset/images/no_course_image.png`